starknet-rpc-test/src/utils.rs

```diff
@@ -0,0 +1,28 @@
+use starknet_accounts::{Account, Call, Execution, SingleOwnerAccount};
+use starknet_core::types::FieldElement;
+use starknet_core::utils::get_selector_from_name;
+use starknet_providers::jsonrpc::{HttpTransport, JsonRpcClient};
+use starknet_signers::LocalWallet;
+
+use crate::constants::FEE_TOKEN_ADDRESS;
+
+pub fn transfer_tokens<'a>(
+    account: &'a SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalWallet>,
```

Comment:
It would be more developer friendly it was implemented in a more OO way:
```rust
alice_account.transfer_tokens();
```

Either by creating a wrapper type over `SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalWallet>` or, even better, through a trait `AccountActions` that would define those methods